AUTHORS.md

```diff
@@ -0,0 +1,4 @@
+Authors
+=======
+- [Antranig Basman](https://github.com/amb26)
+- [Leopold Burdyl](https://github.com/vilnius-leopold)
```

Comment:
@vilnius-leopold, I think that it is fine by just adding yourself after https://github.com/GPII/node-gsettings/blob/master/LICENSE#L1, so you can just get rid of https://github.com/vilnius-leopold/node-gsettings-gpii-integration/blob/master/FORKED_LICENSE.txt